### PR TITLE
Update README to link to recent documentation

### PR DESCRIPTION
**Pull request by thomasnares**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Updates readme to link to recent documentation

### CODE REVIEW

**Comment by thomasnares** [comment]
@kpodemski

**Review by matks — APPROVED** [review]
Looks good to me

Waiting for @kpodemski final decision before merge 😉

**Review by tleon — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

**Comment by matks** [comment]
Given there is 3 approvals I feel safe approving that 👍

**Pull request merged**
